JitsiTrackEvents.ts

```diff
@@ -45,6 +45,16 @@ export enum JitsiTrackEvents {
      */
     NO_AUDIO_INPUT = 'track.no_audio_input',
 
+    /**
+     * Indicates that a new owner has been assigned to a remote track when SSRC rewriting is enabled.
+     */
+    TRACK_OWNER_ADDED = 'track.owner_added',
```

Comment:
"added" and "removed" read weird here. How about a single `TRACK_OWNER_SET` ? We'd only need 1 then.